runner/runner.go

```diff
@@ -2,36 +2,41 @@ package runner
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"fmt"
 	"io"
 	"os"
 	"os/exec"
-	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sfn"
+	"github.com/aws/aws-sdk-go/service/sfn/sfniface"
 
 	"gopkg.in/Clever/kayvee-go.v6/logger"
 )
 
-var log = logger.New("batchcli")
+var log = logger.New("sfncli")
 
 type TaskRunner struct {
-	job    BatchJob
-	store  ResultsStore
-	cmd    string
-	inputs []string
+	sfnapi    sfniface.SFNAPI
+	taskToken string
+	cmd       string
+	inputs    []string
 }
 
 // Process runs the underlying cmd with the appropriate
 // environment and command line params
-func (t TaskRunner) Process() error {
+func (t TaskRunner) Process(ctx context.Context) error {
+	if t.sfnapi == nil {
+		return nil // if New failed :-/
+	}
 	log.InfoD("exec-command", map[string]interface{}{
-		"inputs":       t.inputs,
-		"cmd":          t.cmd,
-		"job-id":       t.job.JobId,
-		"dependencies": strings.Join(t.job.DependencyIds, ","),
+		"inputs": t.inputs,
+		"cmd":    t.cmd,
```

Comment:
of course we will need the appName, job-id here. Also need to figure out a way to get WorkflowID and WorkflowName here if possible

Reply:
it doesn't look like this is possible :-/

Reply:
specifically, I don't see a way that the code here could get either an execution ID or the state machine ID that it's associated with

Reply:
hmmm. this would make figuring out logs that belong to a workflow hard. Maybe we can solve this by having the UX pull out the activityTokens for each Workflow (executionID) and making the appropriate log queries. In that case we should probably log the `activityToken`?

Reply:
activity tokens are super long strings that don't show up in any place in the API other than the polling that the CLI does, so I don't think it's the identifier that will connect things together for a workflow...

one small breakthrough: d0f705a

If you poll with a `workerName`, then that name shows up in the execution history. We could set `workerName` to the container task, for example, and then at least we know which container was part of the workflow. This together with the execution history timestamps would give probably give us a "good enough" search to run on logs data to get all the logs for a workflow.